src/parser/hunter/shared/modules/spells/HuntersMark.tsx

```diff
@@ -48,15 +48,15 @@ class HuntersMark extends Analyzer {
   protected enemies!: Enemies;
   protected abilities!: Abilities;
 
-  constructor(options: any) {
+  constructor(options: Options) {
     super(options);
     this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.HUNTERS_MARK), this.onCast);
     this.addEventListener(Events.removedebuff.by(SELECTED_PLAYER).spell(SPELLS.HUNTERS_MARK), this.onDebuffRemoval);
     this.addEventListener(Events.applydebuff.by(SELECTED_PLAYER).spell(SPELLS.HUNTERS_MARK), this.onDebuffApplication);
     this.addEventListener(Events.refreshdebuff.by(SELECTED_PLAYER).spell(SPELLS.HUNTERS_MARK), this.onDebuffRefresh);
     this.addEventListener(Events.damage.by(SELECTED_PLAYER), this.calculateMarkDamage);
 
-    options.abilities.add({
+    (options.abilities as Abilities).add({
```

Comment:
is this equivalent to `this.abilities` as its added a dep?

Reply:
Yes